tests/test_cms_menus.py

```diff
@@ -75,37 +74,61 @@ def test_get_nodes(self):
 
     def test_get_roots_with_multiple_menucontents(self):
         """test to check get_roots while creating two menu contents"""
-        factories.MenuContentFactory()
-        factories.MenuContentFactory()
+        menucontent_1 = factories.MenuContentFactory()
+        menucontent_2 = factories.MenuContentFactory()
         roots = self.menu.get_roots(self.request)
         self.assertEqual(roots.count(), 2)
+        self.assertListEqual(list(roots), [menucontent_1.root, menucontent_2.root])
 
-
-class CMSMenuVersioningTestCase(CMSTestCase):
-    def setUp(self):
-        self.menu = CMSMenu(None)
-        self.request = RequestFactory().get("/")
-        self.user = self.get_superuser()
-
-        # setting up menu content with versions
-        menuitem_1 = factories.RootMenuItemFactory()
-        menuitem_1_v1 = factories.MenuVersionFactory(content__root=menuitem_1)
-        menuitem_1_v1.archive(self.user)
-        menuitem_1_v1.copy(self.user)
-        menuitem_2 = factories.RootMenuItemFactory()
-        factories.MenuVersionFactory(content__root=menuitem_2)
-
-    def test_get_roots_with_versioning_enabled(self):
+    def test_get_roots_with_versions(self):
         """This test to check versioning would group all the versions
         of menu content and return latest of all distinct menu content
         """
+        menuitem_1 = factories.RootMenuItemFactory()
+        menucontent_1_version_1 = factories.MenuVersionFactory(content__root=menuitem_1)
+        menucontent_1_version_1.archive(self.user)
+        menucontent_1_version_2 = menucontent_1_version_1.copy(self.user)
+        menuitem_2 = factories.RootMenuItemFactory()
```

Comment:
I thought the reason you created the root menu items separately was to be able to assert them more easily below, but then you're not using those vars, so what do you need them for?